mappings/net/minecraft/village/TradeOffers.mapping

```diff
@@ -90,8 +92,22 @@ CLASS net/minecraft/unmapped/C_hqrlptsj net/minecraft/village/TradeOffers
 		METHOD <init> (Lnet/minecraft/unmapped/C_gmbqjnle;IILnet/minecraft/unmapped/C_sddaxwyk;IIIF)V
 			ARG 1 item
 			ARG 2 count
+			ARG 3 price
 			ARG 4 processedItem
 			ARG 5 processedItemCount
+			ARG 6 maxUses
+			ARG 7 experience
+			ARG 8 multiplier
```

Comment:
Multiplier for what? Is there a more descriptive name that makes sense?
(On mobile, can't check source)